src/main/fulcro/tempid.cljc

```diff
@@ -1,6 +1,8 @@
 (ns fulcro.tempid
   #?(:clj (:import [java.io Writer])))
 
+(def tag "fulcro/tempid")
```

Comment:
If you want to add this, I'd prefer it be authoritative...that is, used within the source instead of just declared.

Reply:
done! :)